lib/posts.js

```diff
@@ -220,4 +221,48 @@ PostsRequest.prototype.statuses = function() {
 	return this;
 };
 
+/**
+ * @method year
+ * @chainable
+ * @param year - Integer for year
```

Comment:
The format we've been using for docblocks defines a more rigorous syntax for parameter types with curly brackets: this permits us to auto-generate the documentation. See [the @param](http://yui.github.io/yuidoc/syntax/#param) definition: this could be re-written as

```js
/**
 * @param {Number} year An integer defining the year for which to retrieve posts
 */
```